pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -60,7 +60,7 @@ Please add breaking changes here when merged to the `develop` branch.
 - preciceDt = interface.initialize();
   
 - if(interface.isActionRequired(cowid)){
--   interface.writeBlockVectorData(forceID, vertexSize, vertexIDs, forces);
+-   interface.writeBlockVectorData(forceID, vertexSize, vertexIDs, forces.data());
```

Comment:
This should be `forces` too.

In the example, we change from `double *forces` to `std::vector<double> forces`. The argument used to be a pointer and has changed to a `span<const double>`, so both are correct.

Yes, it is confusing, but it is correct.

```suggestion
-   interface.writeBlockVectorData(forceID, vertexSize, vertexIDs, forces);
```

Reply:
Thanks, now it actually make sense to me. I do think that using `std::vector` in both versions is a more useful example. That we should not use raw pointers is already stated anyways in one of the bullet points
> The signature of `readData`, `writeData` and `writeGradientData` has changed from `const int*`, `const double*`, and `double*` to `span<const VertexID>`, `span<const double>`, and `span<double>`. If necessary change the data object, e.g., from `double* forces = new double[vertexSize*dim]` to `std::vector<double> forces(vertexSize*dim)` and remove `.data()` in the function arguments.